script/hassfest/strict_typing.py

```diff
@@ -0,0 +1,54 @@
+"""Ensure strict_typing file is valid and sorted."""
```

Comment:
I don't think it should be s separate plugin
Can you do it inside the mypy plugin?

Reply:
That's how I started this, but the hassfest architecture guides towards one "plugin" doing one thing only, so it seemed more appropriate to be in a separate module.

Reply:
strict_typing and mypy are directly linked so it seems awkward to have a new plugin.

Let's wait and see what others think.

Reply:
Please look also at the precomit definition

Reply:
> Please look also at the precomit definition

Do you mean https://github.com/home-assistant/core/blob/a255546e9d76a95c7c670a6d6e761f5eefa49801/.pre-commit-config.yaml#L125-L131

(i.e. that changes to `.strict-typing` currently only trigger `mypy_config`)?